docker-compose-dev.yml

```diff
@@ -26,5 +26,4 @@ services:
       - DJANGO_SUPERUSER_USERNAME=admin
       - DJANGO_SUPERUSER_PASSWORD=1234
       - DJANGO_SUPERUSER_EMAIL=admin@pycon.tw
-    entrypoint: /docker-entrypoint.sh
-    command: python /app/src/manage.py runserver 0.0.0.0:8000
```

Comment:
I notice that if someone runs the server through `docker-compose -f ./docker-compose-dev.yml up` the service `pycontw_dev` would exit directly. Should we handle this situation? Maybe adding `command: tail -f /dev/null` here so that people could continue to interact with it?

Reply:
nice idea!